Feature/new component logo

DeepaliAgrwl opened the pull request:
Please merge the changes

DeepaliAgrwl committed:
Logo component changes merged with development

Logo component changes merged with development

DeepaliAgrwl committed:
added extension in logo redirect

added extension in logo redirect

rakeshkumar-verma commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/_cq_dialog/.content.xml:
```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:granite="http://www.adobe.com/jcr/granite/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    jcr:title="Logo"
+    sling:resourceType="cq/gui/components/authoring/dialog"
+    extraClientlibs="[core.wcm.components.image.v2.editor]">
+    <content
+        granite:class="cmp-image__editor"
+        jcr:primaryType="nt:unstructured"
+        sling:resourceType="granite/ui/components/coral/foundation/container">
```
@DeepaliAgrwl  Why you need this dialog here?

DeepaliAgrwl replied:
Not required.removed

DeepaliAgrwl committed:
removed dialog from logo component

removed dialog from logo component

DeepaliAgrwl committed:
Merge branch 'development' into feature/new-component-logo

rakeshkumar-verma commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/clientlib/less/logo.less:
```diff
@@ -0,0 +1,6 @@
+.logo-container{
+
+display : flex;
```
@DeepaliAgrwl  Remove unused styling

rakeshkumar-verma commented on ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/logo/.content.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:Component"
```
@DeepaliAgrwl  also updated header component policy

DeepaliAgrwl committed:
Review Comments incorporated

1. removed clientlibs
2. udated policies
2. added cq_dialog and hode unused functionality from dialog

DeepaliAgrwl committed:
changing dialog title to Logo
